frontend/catalyst/pennylane_extensions.py

```diff
@@ -383,6 +383,21 @@ def __call__(self):
 
         return self._call_with_quantum_ctx(ctx)
 
+    def _call_during_interpretation(self):
+        """Create a callable for conditionals."""
+        if self.pred:
+            return self.true_fn()
+        if self.false_fn:
+            return self.false_fn()
+        return None  # pragma: no cover
```

Comment:
Why this uncovered line?

Reply:
Just created a test case [here](https://github.com/PennyLaneAI/catalyst/pull/31/commits/53832d107a9d65ed44af07c8396e13a47f58b855). Thanks!

Reply:
@dime10 thought I fixed it, but it was actually correct. This line will never be executed because the default value for `self.false_fn` is a lambda that returns `None`. If you want I can revert https://github.com/PennyLaneAI/catalyst/pull/31/commits/53832d107a9d65ed44af07c8396e13a47f58b855.